### ISSUE TITLE
Unable to view authentication settings with 4.0.0-rc.1-2

### ISSUE THREAD

**Issue by krya-kryak** [Graylog2/graylog2-server#9481]
After upgrading graylog from 4.0-beta4 to 4.0.0-rc.1-2 I am unable to view authentication settings.

## Expected Behavior
/system/authentication/services/active opens without issues

## Current Behavior
/system/authentication/services/active displays an error page "Something went wrong":
```
Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.


Stack Trace:

Cc@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:333543
ko/<@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:279681
Ci@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:289695
Ls@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:328319
Mc@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:320633
mc@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:320556
sc@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:317586
Za/<@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:269227
434/exports.unstable_runWithPriority@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:780503
Ua@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:268936
Za@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:269174
Va@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:269107
nc@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:314371
yi@https://HOSTNAME/assets/vendor.594b2a39cb22b445205e.js:2:286185
s@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2452058
o/<[22]</n.exports/T.prototype._settlePromiseFromHandler@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2424985
o/<[22]</n.exports/T.prototype._settlePromise@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2425785
o/<[22]</n.exports/T.prototype._settlePromise0@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2426484
o/<[22]</n.exports/T.prototype._settlePromises@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2427836
l@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2378997
c@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2378937
o/<[2]</a.prototype._drainQueues@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2380051
a/this.drainQueues@https://HOSTNAME/assets/app.031d0681dddd597e4deb.js:2:2378870


Component Stack:

    in ot
    in dt
    in div
    in div
    in SectionGrid
    in B
    in i
    in Unknown
    in Unknown
    in n
    in t
    in t
    in div
    in t
    in div
    in t
    in AppWithoutSearchBar__StyledRow
    in div
    in t
    in div
    in AppContentGrid__Container
    in u
    in f
    in t
    in t
    in div
    in Lt
    in o
    in dt
    in h
    in Rt
    in t
    in t
    in c
    in t
    in Ir
    in Unknown
    in ConnectStoresWrapper[Unknown/Anonymous] stores=streams
    in je
    in T
    in f
    in v
    in b
    in Unknown
    in n
    in F
    in ConnectStoresWrapper[F] stores=currentUser,server,sessionId
    in je
    in T
    in Z
```

## Steps to Reproduce (for bugs)
1. Set up graylog with docker image graylog/graylog:4.0.0-rc.1-2 behind SSO proxy
2. Enable LDAP integration with SSO header authenticator
3. Navigate to System / Authentication -> Authentication
4. Wait for a page to load, get a banana peel on a mokey with a react error

## Context
I am unable to view / edit authentication settings.

* Graylog Version: graylog/graylog:4.0.0-rc.1-2
* Operating System: MacOS 10.15.7 Catalina
* Browser version: seems to be irrelevant, tried Firefox 82.0.3 (64-bit) and Safari Version 14.0 (15610.1.28.1.9, 15610)

**Comment by bernd** [comment]
@krya-kryak Thank you for the bug report! We are able to reproduce it locally and are working on a fix.

**Comment by bernd** [comment]
@krya-kryak This will be fixed in the upcoming 4.0.0-rc.2 release which we will build today. Thanks again for testing the pre-release and reporting the issue! :+1:
